gensrc/thrift/PlanNodes.thrift

```diff
@@ -573,8 +574,9 @@ struct TAggregationNode {
   // used in query cache
   25: optional list<Exprs.TExpr> intermediate_aggr_exprs
 
+  26: optional bool use_sort_agg
   // used in pipeline engine
-  26: optional bool interpolate_passthrough = false
+  27: optional bool interpolate_passthrough = false
```

Comment:
for compatibility, please add the new one at the last.

  26: optional bool interpolate_passthrough = false
  27: optional bool use_sort_agg